astropy/modeling/tests/test_constraints.py

```diff
@@ -184,6 +183,7 @@ def test_bounds_slsqp(self):
         assert intercept + 10**-5 >= bounds["intercept"][0]
         assert intercept - 10**-5 <= bounds["intercept"][1]
 
+    @pytest.mark.filterwarnings("ignore:The fit may be unsuccessful")
```

Comment:
@WilliamJamieson , if you are not okay with this, please open follow-up PR. Thanks!